components/desktop/KnitProjectsWidget.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <section class="projects-widget">
+    <h2 class="projects-widget__title">
+    <span class="projects-widget__title-icon flaticon-code-fork" aria-hidden="true"></span>
+      Projekty do których możesz dołączyć
+    </h2>
+    <div v-for="item of items" class="projects-widget__projects-wrapper">
+      <knit-projects-widget-item :title="item.title" :maxTeamSize="item.maxTeamSize"
+        :currentTeamSize="item.currentTeamSize" :repoLink="item.repoLink" :tags="item.tags" :projectLogoSource="item.projectLogoSource"/>
+    </div>
+    <span class="projects-widget__see-more">
+      Zobacz więcej
+    </span>
+  </section>
+</template>
+
+<script>
+import KnitProjectsWidgetItem from '~/components/desktop/partials/KnitProjectsWidgetItem'
+
+export default {
+  data () {
+    return {
+      items: [
+        {
+          title: 'homepage-nuxtjs',
+          maxTeamSize: 10,
+          currentTeamSize: 3,
+          repoLink: 'https://github.com/knit-pk/homepage-nuxtjs',
+          tags: ['PHP', 'JavaScript', 'Nuxt.js', 'Vue.js'],
+          projectLogoSource: 'images/green-star.png'
+        },
+        {
+          title: 'knit-api',
+          maxTeamSize: 8,
+          currentTeamSize: 5,
+          repoLink: 'https://github.com/knit-pk/api-v1-php',
+          tags: ['PHP', 'Docker'],
+          projectLogoSource: 'images/teal-star.png'
+        },
+        {
+          title: 'knit-homepage-panel',
+          maxTeamSize: 20,
+          currentTeamSize: 3,
```

Comment:
If there is team, maybe add array of members

Reply:
For what?